cmds.pl

```diff
@@ -21,6 +21,8 @@ sub task_add {
     &draw_prompt_line('');
     return;
   }
+  # TODO: fix this evilness
```

Comment:
The proper way to solve this would probably be to use IPC::Open2 in task_exec() instead of a regular open() with shell expansion.  Using open2() would allow you to avoid the shell altogether (with all its escaping and expansion pitfalls) and as a bonus allows you to specify each command line argument for ``task`` separately.

However, fixing this is quite some work, I think, as it would require rewriting not only the ``task_exec()`` function, but also large parts of ``cmds.pl``.